sources/Launcher/DYNMarginCalculationLauncher.cpp

```diff
@@ -773,25 +765,24 @@ MarginCalculationLauncher::findOrLaunchLoadIncrease(const boost::shared_ptr<Load
   for (unsigned int i = 0; i < variationsToLaunch.size(); i++) {
     auto currVariation = variationsToLaunch.at(i);
     loadIncreaseStatus_.insert(std::make_pair(currVariation, LoadIncreaseStatus(allSuccesses.at(i))));
-    auto loadIncreaseResult = importResult(computeLoadIncreaseScenarioId(currVariation));
-    TraceInfo(logTag_) << DYNAlgorithmsLog(LoadIncreaseEnd, currVariation, getStatusAsString(loadIncreaseResult.getStatus())) << Trace::endline;
+    const SimulationResult importedResult2 = importResult(computeLoadIncreaseScenarioId(currVariation));
```

Comment:
Why changing the variable name?

Reply:
We decided to pass a LoadIncreaseResult object in the MarginCalculationLauncher::findOrLaunchLoadIncrease() method, so I felt like loadIncreaseResult wasn't appropriate for this variable.

Reply:
it is still the load increase result